### ISSUE TITLE
Warning about Mako after restore

### ISSUE THREAD

**Issue by NicoLeOca** [pymedusa/SickRage#1056]
I just deleted and restore Medusa
and I had this:

2016-09-09 08:02:58 WARNING  MAIN :: [34ff295] Restore: Unable to remove the cache/mako directory: error 2 : No such file or directory

I think I get this every single time I restore.

Any idea why please?

full logs:
https://gist.github.com/NicoLeOca/aa85af73b5cbe881d3a1a81b8fab3b01

**Comment by fernandog** [comment]
@NicoLeOca how old was the backup?

**Comment by fernandog** [comment]
@medariox this is really weird:

code checks if path exists but the error is `No such file or directory`
maybe permission?

he is running DSM and I think he can't change app files

```python
            if ek(os.path.exists, ek(os.path.join, DATA_DIR, 'restore')):
                try:
                    shutil.rmtree(ek(os.path.join, DATA_DIR, 'restore'))
                except Exception as e:
                    logger.log(u"Restore: Unable to remove the restore directory: {0}".format(ex(e)), logger.ERROR)
```

**Comment by fernandog** [comment]
@ratoaq2 do you have DSM too right? can you confirm Medusa can't remove app files?

**Comment by ratoaq2** [comment]
@NicoLeOca 

Could you list the files permissions under cache folder inside medusa folder?

```
ls -lthaR /usr/local/sickbeard-custom/var/cache
```

**Comment by NicoLeOca** [comment]
@ratoaq2 
https://gist.github.com/NicoLeOca/fb6586dac9ea798d5d4bcc1b5513eef2

**Comment by NicoLeOca** [comment]
@fernandog 
sorry I missed your question.
my backup was fresh
I had to do uninstall and reinstall to pass over issue #1048

**Comment by ratoaq2** [comment]
The actual issue is not [here](https://github.com/pymedusa/SickRage/blob/master/sickbeard/__init__.py#L797)

The issue is [here](https://github.com/pymedusa/SickRage/blob/master/sickbeard/__init__.py#L804): 

Just look at this log line (it says cache/mako, and not restore):
```
2016-09-09 08:02:58 WARNING  MAIN :: [34ff295] Restore: Unable to remove the cache/mako directory: error 2 : No such file or directory
```

**Comment by ratoaq2** [comment]
Fixed by #1080

**Comment by NicoLeOca** [comment]
@ratoaq2 
thank you
